src/main/java/com/feywild/feywild/events/loot_modifiers/MineshaftChestLootModifier.java

```diff
@@ -15,37 +13,36 @@
 import org.jetbrains.annotations.NotNull;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MineshaftChestLootModifier extends LootModifier {
 
-    private final Item addition;
-
-    protected MineshaftChestLootModifier(LootItemCondition[] conditionsIn, Item addition) {
+    protected MineshaftChestLootModifier(LootItemCondition[] conditionsIn) {
```

Comment:
Little hint: Try to avoid to copy variable names that end in `In` or start with a lowercase `p` followed by an uppercase letter. Both things have been applied to vanilla parameters to make the minecraft code recompilable after forge patches. So in this case, Just name the variable `conditions`. I won't add a commen tverywhere, just try to remember that if possible.